opentelemetry-kotlin-compat/src/jvmTest/kotlin/io/embrace/opentelemetry/kotlin/k2j/logging/LogExportTest.kt

```diff
@@ -58,4 +67,47 @@ internal class LogExportTest {
             sanitizeSpanContextIds = true,
         )
     }
+
+    @Test
+    fun `test context is passed to processor`() {
+        // Create a LogRecordProcessor that captures any passed Context.
+        val contextCapturingProcessor = ContextCapturingProcessor()
+
+        // Use it on OpenTelemetryInstance creation.
+        val kotlinApi = OpenTelemetryInstance.kotlinApi(
+            clock = FakeClock(),
+            loggerProvider = {
+                addLogRecordProcessor(contextCapturingProcessor)
+            }
+        )
+
+        // Create a context key and add a test value
+        val rootContext = Context.Companion.current()
+        val contextKey = rootContext.createKey<String>("best_team")
+        val testContextValue = "independiente"
+        val testContext = rootContext.set(contextKey, testContextValue)
+
+        // Log a message with the created context
+        val logger = kotlinApi.loggerProvider.getLogger("test_logger")
+        logger.log(body = "Test log with context", context = testContext)
+
+        // Verify context was captured and contains expected value
+        val actualValue = contextCapturingProcessor.capturedContext?.get(contextKey)
+        assertEquals(testContextValue, actualValue)
```

Comment:
`assertSame` would be a stronger assertion here as the reference _should_ be the same (unless there's wrappers getting in the way that mean it isn't the same exact instance)